docs/getting-started/quickstart/README.md

```diff
@@ -383,20 +385,20 @@ code ./sample_append.js
 Alternatively, you can locate and open the file from the EXPLORER window on the left of Codespaces. You can find the path to the file in the command above.
 :::
 
-2. In step 1 of the code, review how the client connects to EventStoreDB:
+2. In step 1 of the code, review how the client connects to KurrentDB:
 
 ::: tabs#dev-language-platform
 @tab Select >
 
 @tab Python
 ```python
-# Create an instance of EventStoreDBClient, connecting to the EventStoreDB at localhost without TLS
+# Create an instance of EventStoreDBClient, connecting to the KurrentDB at localhost without TLS
 client = EventStoreDBClient(uri="esdb://localhost:2113?tls=false")  
```

Comment:
The connection string will need to be updated for all of these:
```suggestion
client = EventStoreDBClient(uri="kurrentdb://localhost:2113?tls=false")  
```
And is `EventStoreDBClient` correct here, or has that been renamed to `KurrentDBClient`?

Reply:
Ideally code snippets should not be hardcoded here. Since this is the only file, I'll adjust them directly.